modules/decryption.js

```diff
@@ -0,0 +1,52 @@
+import { joinArrayAsString } from '../helpers';
+import { ENCRYPTED_NUMBERS } from '../constants/narcissistNumbers';
+
+const MAX_TOKEN = 25;
+
+let KEY;
+
+function convertEncryptedStringToMinTokenBlocks(str) {
+	const string = str.trim();
+	let i = 0;
+	const resultArray = [];
+	const encryptedNumberValues = Object.values(ENCRYPTED_NUMBERS);
+
+	while (i < string.length) {
+		const stringPlusOne = string[i] + string[i + 1];
+		if (encryptedNumberValues.indexOf(stringPlusOne) >= 0) {
+			resultArray.push(stringPlusOne);
+		}
+		if (numPlusOne > MAX_TOKEN || i === string.length - 1) {
+			resultArray.push([string[i]]);
+			i = i + 1;
+		} else {
+			const numString = string[i] + string[i + 1];
+			resultArray.push([numString]);
+			i = i + 2;
+		}
```

Comment:
Please break into smaller functions. Logic is becoming convoluted

Reply:
Removing the file itself. It is a decryption which I believe we are not doing at this point